soh/soh/OTRGlobals.cpp

```diff
@@ -673,51 +687,55 @@ extern "C" uint16_t ResourceMgr_LoadTexHeightByName(char* texPath);
 extern "C" uint32_t ResourceMgr_LoadTexSizeByName(const char* texPath);
 
 extern "C" char* ResourceMgr_LoadTexOrDListByName(const char* filePath) {
-    auto res = OTRGlobals::Instance->context->GetResourceManager()->LoadResource(filePath);
+    auto res = ResourceMgr_LoadResource(filePath);
 
     if (res->resType == Ship::ResourceType::DisplayList)
         return (char*)&((std::static_pointer_cast<Ship::DisplayList>(res))->instructions[0]);
     else if (res->resType == Ship::ResourceType::Array)
         return (char*)(std::static_pointer_cast<Ship::Array>(res))->vertices.data();
-    else
-        return ResourceMgr_LoadTexByName(filePath);
+    else {
+        std::string Path = filePath;
+        if (ResourceMgr_IsGameMasterQuest()) {
+            size_t pos = 0;
+            if ((pos = Path.find("/nonmq/", 0)) != std::string::npos) {
+                Path.replace(pos, 7, "/mq/");
```

Comment:
Needed to also intercept for MQ here, as the `LoadTexByName` by function is in libultraship, so shouldn't modify it for MQ specifically.